### PR TITLE
🚚 update: Add 106th CatchUp Summary

### PR DESCRIPTION
**Pull request by PranavDani**

### CODE REVIEW

**Review by tusharnankani — APPROVED** [review]
Amazing! LGTM.

File: summary/sessions/106/content.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* link:https://twitter.com/chiranjeevVsri[Chiranjeev Srivastava^] shared Holopin tags and created a new GitHub repository for showcasing them on his profile.
+* link:https://twitter.com/furtado_jaden[Jaden Furtado^] unknowingly ran a FUD (Fully Undetectable) malware multiple times thinking that it was a library for his use case. 
+    ** The malware essentially steals files from the host machine and sends them to a remote server.
+    ** Link for the virus -> link:https://github.com/joewalnes/websocketd[Websocketd^] (Do not run ⚠️).
+    ** link:https://twitter.com/SirusTweets[Darshan Rander^] modified the wiki for the repository with the malicious links.
+* We discussed how doing assignments diligently could be beneficial for most students.
+* link:https://twitter.com/SirusTweets[Darshan Rander^] and link:https://twitter.com/t3_pat[Pratik Thakare^] shared that they were working on a WhatsApp chat analysis web app using link:https://nuxtjs.org[Nuxt 3^] for the front end and link:https://www.rust-lang.org[Rust^] for its backend which is supposed to compile in link:https://webassembly.org[Wasm^] and run on the client side to reduce server load.
```
Comment (SirusCodes) [inline summary/sessions/106/content.adoc:16]:
```suggestion
* link:https://twitter.com/SirusTweets[Darshan Rander^] and link:https://twitter.com/t3_pat[Pratik Thakare^] shared that they were working on a WhatsApp chat analysis web app using link:https://nuxt.com[Nuxt 3^] for the front end and link:https://www.rust-lang.org[Rust^] for its backend which is supposed to compile in link:https://webassembly.org[Wasm^] and run on the client side to reduce server load.
```

File: summary/sessions/106/content.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    ** They took reference from Tushar's WhatsApp chat analysis project.
+        *** link:https://medium.com/@tusharnankani/an-exhaustive-whatsapp-chat-data-analysis-f8421a845c30[Medium Article^]
+        *** link:https://github.com/tusharnankani/whatsapp-chat-data-analysis[GitHub Repo^]
+* We talked about the working behind link:https://webassembly.org[Wasm^]. 
+    ** WebAssembly (Wasm) is a binary instruction format for a stack-based virtual machine.
```
Comment (SirusCodes) [inline summary/sessions/106/content.adoc:21]: I think it should be all caps
```suggestion
* We talked about the working behind link:https://webassembly.org[WASM^]. 
    ** WebAssembly (WASM) is a binary instruction format for a stack-based virtual machine.
```

**Review by SirusCodes — CHANGES_REQUESTED** [review]

File: summary/sessions/106/content.adoc (hunk 2 above)
Comment (PranavDani) [reply summary/sessions/106/content.adoc:21]: It was all caps initially, but I changed it because it wasn't caps in the docs.

Comment (PranavDani) [reply summary/sessions/106/content.adoc:21]: "WebAssembly (abbreviated Wasm)"
 https://webassembly.org/#:~:text=WebAssembly%20(abbreviated%20Wasm)

**Pull request merged**